source/_integrations/fritz.markdown

```diff
@@ -37,7 +37,7 @@ There is support for the following platform types within Home Assistant:
 {% include integrations/config_flow.md %}
 
 <div class='note'>
-TR-064 needs to be enabled in the FRITZ!Box network settings for Home Assistant to login and read device info.
+Both TR-064 and UPnP need to be enabled in the FRITZ!Box ( Home Network -> Network -> Network settings -> Access Settings in the Home Network ) for Home Assistant to login and read device info.
```

Comment:
Fritzbox login and (many, not all) sensors are available without the UPnP service. E.g. presence detection based on wifi/ethernet connection, router connection uptime work without it. Current download and upload speed (and some more) only with UPnP Status enabled.

Suggestion:

> TR-064 needs to be enabled in the FRITZ!Box network settings for Home Assistant to login and read device info. UPnP need to be enabled in the FRITZ!Box (Home Network -> Network -> Network settings -> Access Settings in the Home Network) for Home Assistant to read additional device info such as current download throughput.

Reply:
Not sure we should go that way as not enabling it is causing issues.

For sure we can check if it's enabled or not and act accordingly, but official AVM documentation says that they must be both enabled to make TR064 fully working, so...

Reply:
I get your thumb up as a ok, so I mark this conversation as resolved.